docs/src/examples/components/Toolbar/Types/ToolbarExampleEditor.shorthand.tsx

```diff
@@ -129,7 +131,36 @@ const ToolbarExampleShorthand = () => {
         { key: 'bullets', icon: { name: 'bullets', outline: true } },
         { key: 'number-list', icon: { name: 'number-list', outline: true } },
         { key: 'divider3', kind: 'divider' },
-        { key: 'more', icon: { name: 'more', outline: true } },
+        {
+          key: 'more',
+          icon: { name: 'more', outline: true },
+          active: moreMenuOpen,
+          menu: {
```

Comment:
I believe that `items` are redundant there:

```jsx
menu: [{ item }, { item }]
```

As we have `mappedArrayProp` to handle this